airflow-core/src/airflow/api_fastapi/common/types.py

```diff
@@ -96,3 +96,10 @@ class MenuItem(Enum):
     PROVIDERS = "Providers"
     VARIABLES = "Variables"
     XCOMS = "XComs"
+
+
+class UIAlert(BaseModel):
```

Comment:
```suggestion
class UIAlertResponse(BaseModel):
```

To follow the convention of other serializers, also that should go into `config.py` datamodels, that's not a type but a serializer and I don't think it's re-used at other places.

Reply:
Ack